<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Система меддиагностики</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt corbel;
background: rgb(245, 245, 245);	</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>671</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget QWidget {
	background: rgb(245, 245, 245);
	color:black;
}
QTabWidget::pane {
  border: none;
} 

QTabBar::tab{
	background: red;
	width:0; 
	height: 0; 		
	margin: 0;
	padding: 0;
	border: none;
} 
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="startAnalyze">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string/>
     </attribute>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditSearch">
            <property name="styleSheet">
             <string notr="true">background-color: white;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSearch">
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tablePatient">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>16</number>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonDownFile">
          <property name="text">
           <string>Далее</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonAddPatient">
          <property name="text">
           <string>Добавить пациента</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QFormLayout" name="formLayout">
        <property name="verticalSpacing">
         <number>16</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelSurname_2">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="surnamePatient">
          <property name="styleSheet">
           <string notr="true">background-color: white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelName_2">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="namePatient">
          <property name="styleSheet">
           <string notr="true">background-color: white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelPatronymic_2">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="patronymicPatient">
          <property name="styleSheet">
           <string notr="true">background-color: white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelDate">
          <property name="text">
           <string>Дата рождения</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="datePatient">
          <property name="styleSheet">
           <string notr="true">background-color: white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelGender">
          <property name="text">
           <string>Пол</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="genderPatient">
          <property name="styleSheet">
           <string notr="true">background-color: white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="downloadFile">
     <attribute name="title">
      <string/>
     </attribute>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labelFilePath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonLoadFile">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Загрузить файл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAnalyze">
          <property name="text">
           <string>Далее</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="index1ButtonBack">
          <property name="text">
           <string>назад</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Analyze">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QProgressBar" name="progressAnalyze">
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>Waiting...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>390</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="buttonResult">
         <property name="text">
          <string>Далее</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="index2ButtonBack">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="buttonStartAnalyze">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Анализ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resultAnalyze">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QTextEdit" name="textEditResult">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>391</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Результаты анализа</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>390</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="saveDB">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Cохранить в бд</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="index3ButtonBack">
         <property name="text">
          <string>назад</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="addPatient">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>581</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelSurname">
         <property name="text">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditSurname"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelPatronymic">
         <property name="text">
          <string>Отчество</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditPatronymic"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelBirthDate">
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="4" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonAddPatientInDB">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="index4ButtonBack">
           <property name="text">
            <string>назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>281</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 211);</string>
    </property>
    <property name="title">
     <string>Пользователь</string>
    </property>
    <widget class="QPushButton" name="buttonExit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>выход</string>
     </property>
    </widget>
    <widget class="QLabel" name="User">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="text">
      <string>Авдеев И.А</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tabBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black; font: 10pt</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="buttonClose">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>5</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonInfo">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>5</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
